### DISCUSSION TITLE
porting tiny pilot for iMX8M Mini.

### DISCUSSION THREAD

**Discussion by sujaybane** [tiny-pilot/tinypilot#951] [Q&A] [answered]
Hello,

I am trying to port a tiny pilot on the iMX8M Mini board using Yocto.
I have referred your GitHub Repository https://github.com/tiny-pilot/tinypilot
wanted to know if it is possible to port Tiny pilot for iMX8M Mini.
if yes, can you suggest to me what changes need to do if I am using the GitHub source code to build for iMX8M Mini?
also, using GitHub source code I have built for my native machine and am able to access the tiny pilot browser page using the localhost:8000 port.
trying the same for iMX Board. if you have any way to port please let me know.

Thanks.

**Comment by mtlynch** [comment, marked as answer]
Unfortunately, it doesn't seem like anyone else has experience with this board. Please keep the discussion to the [alternate hardware thread](https://github.com/tiny-pilot/tinypilot/discussions/755) and refrain from starting new threads about the same topic. Further duplicate posts will result in a ban.
